### PR TITLE
Combine all code into a single module

### PR DESCRIPTION
**Pull request by stevemessick**
Rearrange things so there is a single module that works for IntelliJ and Android Studio.

@pq @devoncarew

### CODE REVIEW

**Comment by stevemessick** [comment]
PTAL. All checks pass now, but I always like someone to double-check code that uses reflection.

File: src/io/flutter/FlutterStudioInitializer.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    }
+    actionManager.registerAction(actionId, newAction);
+  }
+
+  public static void runActivity(@NotNull Project project) {
+    try {
+      @SuppressWarnings("unchecked")
+      Class<AnAction> clazz = (Class<AnAction>)Class.forName("com.intellij.ide.actions.NewProjectAction");
+      AnAction newProject = clazz.newInstance();
+      Method method = clazz.getMethod("getTemplatePresentation");
```
Comment (devoncarew) [inline src/io/flutter/FlutterStudioInitializer.java]: Could you simplify this as `Presentation present = newProject. getTemplatePresentation()`?

**Comment by devoncarew** [comment]
lgtm for the reflection :) thanks for the refactor

File: src/io/flutter/FlutterStudioInitializer.java (hunk 1 above)
Comment (stevemessick) [inline src/io/flutter/FlutterStudioInitializer.java]: Yes, I'll do that then commit.

**Pull request merged**